Backend/UnitTesting/app.config

```diff
@@ -1,40 +1,25 @@
-﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Why do we have two app.config files in UnitTesting?

Reply:
Probably because my computer is dumb.

Reply:
Actually master has two of these files as well, so I don't know what the issue is.

Reply:
I think the different versions of VS running on people's computer does this. One is App.config and the other is app.config. I guess combine them?